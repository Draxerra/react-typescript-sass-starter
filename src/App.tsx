import { FC } from "react";

import store from "./store";
import { Provider } from "react-redux";

import Counter from "./components/Counter";

const App: FC = () => {
  return (
    <Provider store={store}>
      <Counter />
    </Provider>
  );
};

export default App;
