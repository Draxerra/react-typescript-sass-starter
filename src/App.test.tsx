import App from "./App";
import { render } from "@testing-library/react";

describe("App", () => {
  it("should match snapshot", () => {
    const app = render(<App />).asFragment();
    expect(app).toMatchSnapshot();
  });
});
