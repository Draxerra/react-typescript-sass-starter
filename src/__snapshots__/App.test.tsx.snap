// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="app-template"
  >
    <nav
      class="app-nav"
    >
      <a
        aria-current="page"
        aria-label="Navigate to Home Page"
        class="app-link-button app-link-button--active app-button"
        href="/"
      >
        Home
      </a>
      <a
        aria-label="Navigate to Contact Page"
        class="app-link-button app-button"
        href="/contact"
      >
        Contact
      </a>
    </nav>
    <main
      class="app-template__main"
    >
      <div
        class="app-home"
      >
        <div
          class="app-counter"
        >
          <span
            class="app-counter__text"
          >
            Current Count: 0
          </span>
          <button
            aria-label="Increment value"
            class="counter-increment app-button"
          >
            Increment
          </button>
          <button
            aria-label="Decrement value"
            class="counter-decrement app-button"
          >
            Decrement
          </button>
        </div>
      </div>
    </main>
  </div>
</DocumentFragment>
`;
